### ISSUE TITLE
Standard cards being created with weird characters

### ISSUE THREAD

**Issue by CollinHeist** [CollinHeist/TitleCardMaker#65]
Cards with the words `of`, `on`, `the`, `in`, and `at` (maybe others) are being modified like:

![3](https://user-images.githubusercontent.com/17693271/162643503-1df29178-030e-46cf-82e2-6165430f193b.jpg)

This is accompanied with an error like:

```console
error/convert.c/ConvertImageCommand/3322.
composite: unable to open image 'C:\TitleCardMaker\modules\.objects\gradient_title.png': No such file or directory @ error/blob.c/OpenBlob/3527.
composite:  `C:\TitleCardMaker\archives\iZombie (2015)\Generic Season Titles, Generic Font\Season 1\iZombie (2015) - S01E12.jpg' @ error/composite.c/CompositeImageCommand/1634
```

**Comment by CollinHeist** [comment]
I believe the end-result of these odd characters was a combination of the poorly formatted newline characters that caused IM to modify the font.

Still a bit unclear to me, but the person who I was working through this issue with said the fixes of #66 resolved their issues. Will keep closed unless it pops up again

**Comment by CollinHeist** [comment]
Okay, problem was not related to poorly formatted newlines.

These weird ligatures for some characters are a *feature* of the Sequel Neue font (see [here](https://www.dafont.com/sequel.font)), and happens for LOWERCASE text.

I consider this.. not a bug, because the default font case is `upper`, so this is a non-issue, but should be documented for the fixer (when that's done).
